### PR TITLE
[tests] Clear docker volumes on helix

### PR DESCRIPTION
**Pull request by radical**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/5064)

### CODE REVIEW

**Comment by radical** [comment]
This is needed on the build machine too, but I'm not sure if that's OK to do!

File: tests/helix/send-to-helix-inner.proj [hunk 1]
```diff
@@ -105,9 +106,13 @@
     <HelixPreCommand Condition="'$(HasDocker)' == 'true'" Include="docker ps" />
     <HelixPreCommand Condition="'$(HasDocker)' == 'true'" Include="docker container ls --all" />
     <HelixPreCommand Condition="'$(HasDocker)' == 'true'" Include="$(_ShutdownDockerContainersCommand)" />
+    <HelixPreCommand Condition="'$(HasDocker)' == 'true'" Include="docker volume ls" />
+    <HelixPreCommand Condition="'$(HasDocker)' == 'true'" Include="$(_DeleteDockerVolumesCommand)" />
 
     <HelixPostCommand Condition="'$(HasDocker)' == 'true'" Include="docker container ls --all" />
     <HelixPostCommand Condition="'$(HasDocker)' == 'true'" Include="$(_ShutdownDockerContainersCommand)" />
+    <HelixPostCommand Condition="'$(HasDocker)' == 'true'" Include="docker volume ls" />
+    <HelixPostCommand Condition="'$(HasDocker)' == 'true'" Include="$(_DeleteDockerVolumesCommand)" />
```
Comment (eerhardt) [inline tests/helix/send-to-helix-inner.proj:115]: Why are we doing this twice?

Comment (radical) [reply tests/helix/send-to-helix-inner.proj:115]: Pre, and post tests run.

File: tests/Aspire.Workload.Tests/EmptyTemplateRunTests.cs [hunk 2]
```diff
@@ -17,7 +17,6 @@ public EmptyTemplateRunTests(EmptyTemplateRunFixture fixture, ITestOutputHelper
     }
 
     [Fact]
-    [ActiveIssue("https://github.com/dotnet/aspire/issues/4623", typeof(PlaywrightProvider), nameof(PlaywrightProvider.DoesNotHavePlaywrightSupport))]
```
Comment (eerhardt) [inline tests/Aspire.Workload.Tests/EmptyTemplateRunTests.cs:20]: Are these changes supposed to be here?

Comment (radical) [reply tests/Aspire.Workload.Tests/EmptyTemplateRunTests.cs]: Oops, my bad! Fixed now.

**Review by eerhardt — APPROVED** [review]

**Pull request merged**